components/com_users/src/Controller/UserController.php

```diff
@@ -106,7 +107,10 @@ public function login()
         $this->app->setUserState('users.login.form.data', []);
 
         // Show a message when a user is logged in.
-        $this->app->enqueueMessage(Text::_('COM_USERS_FRONTEND_LOGIN_SUCCESS'), 'message');
+        if (ComponentHelper::getParams('com_users')->get('frontend_message', 0))
+        {
```

Comment:
```suggestion
        if (ComponentHelper::getParams('com_users')->get('frontend_message', 0)) {
```
Please use PSR-12 code style. Currently PHPCS fails in drone because of this.

Reply:
Thanks